### PR TITLE
unneeded variable assignments

### PR DESCRIPTION
**Pull request by RaananW**
Once you set public/private in a TypeScript constructor, it is set as this.[variableName] in the constructed class. So there is no need to assign this.amiplifyService, as it is already set.

### CODE REVIEW

File: js/ionic.md [hunk 1]
```diff
@@ -108,9 +108,7 @@ import { AmplifyService }  from 'aws-amplify-angular';
 
 export class AppComponent {
   
-  constructor( public amplify:AmplifyService ) {
```
Comment (mlabieniec) [inline js/ionic.md]: this would need to be defined as private in order to accessible in the other methods

File: js/ionic.md [hunk 2]
```diff
@@ -140,8 +138,6 @@ import { AmplifyService }  from 'aws-amplify-angular';
   // ...
 constructor( public amplifyService: AmplifyService ) {
 
-    this.amplifyService = amplifyService;
```
Comment (mlabieniec) [inline js/ionic.md]: we probably should update this to be a private injection and reference it plainly here

Comment (RaananW) [reply js/ionic.md]: I doubt want to change the visibility of the variable, and I actually thought it should start public so not to break different projects unexpectedly. I'll change it to private

**Comment by jordanranz** [comment]
Docs have been updated with this change already.

Thanks for contributing.

**Pull request closed without merging**